infor-design/enterprise
# 4858/4859 - Fix keyboard for sort and select all from header with datagrid

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixed an issue where the keyboard was not working to sort data for sortable columns and another is select all rows from header checkbox with datagrid

**Related github/jira issue (required)**:
Closes #4858
Closes #4859

**Steps necessary to review your pull request (required)**:
Pull this branch and build/run the demo app

Test 4858
- Navigate to:
- http://localhost:4000/components/datagrid/test-manual-sort.html
- http://localhost:4000/components/datagrid/example-filter.html
- Click on 1st cell in any sort able column
- Use arrow up key to get in header
- Press `space` or `enter` key
- See table data should be sort
- If header has filter, use tab key to navigate to filter-dropdown or filter-input

Test 4859
- Navigate to: http://localhost:4000/components/datagrid/example-mixed-selection.html
- Click on 1st cell in 1st column (checkbox)
- Use arrow up key to get in header checkbox
- Press `space` or `enter` key
- See it should select/unselect all rows

**Included in this Pull Request**:
